CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Smoother scrolling for touchpads (also affects scrolling with some mice that send fractional scrolling values)
```

Comment:
It should be stated that the multiplier now affects mouse scrolling too.

Reply:
It seems the multiplier already affects both types of scrolling in the original code

Reply:
Only for faux scrolling and mouse scrolling. The multiplier shouldn't affect the normal scrolling with touchpads.

You can see that we pass a static value of 1 as a multiplier here:
https://github.com/jwilm/alacritty/blob/master/src/input.rs#L642